src/amo/components/Categories/index.js

```diff
@@ -48,8 +48,8 @@ type Props = {|
 
 type InternalProps = {|
   ...Props,
+  categories: Array<string>,
   categoriesState: $PropertyType<CategoriesStateType, 'categories'>,
```

Comment:
You don't need this as an InternalProp anymore. It can also be removed from `mapStateToProps`.

Reply:
@bobsilverberg Sorry, for not replying on this question. But if I remove the categories from `mapStateToProps`, how will I get the `categories` in the component ?
Also, I want to ask that why we are removing it from `InternalProps`?  I found some error in here, instead of `Array<string>` it should `Array<CategoryType>`?